project/dexter-monitor/public/ctrl/currentCtrl.js

```diff
@@ -25,6 +25,63 @@
  */
 "use strict";
 
-monitorApp.controller("CurrentCtrl", function() {
+monitorApp.controller("CurrentCtrl", function($scope, $http, $log, ProjectService, uiGridConstants) {
     let current = this;
```

Comment:
We use 'controller as' to distinguish among controllers.
If you use only one controller in this file, you can also use just $scope.
Or if you want use 'this' keyword, the variable name can be followed by Controller name.
For example, if controller name is "CurrentCtrl", variable name can be 'currentCtrl'.
refer to https://docs.angularjs.org/api/ng/directive/ngController